platform-sdk/platform-apps/demos/HashgraphDemo/src/main/java/com/swirlds/demo/hashgraph/HashgraphDemoMain.java

```diff
@@ -427,11 +426,7 @@ public void init(final Platform platform, final NodeId id) {
         eventLimit = new TextField(parameters.length <= p ? "" : parameters[p].trim(), 5);
         p++;
 
-        slowCheckbox.addItemListener(new ItemListener() {
-            public void itemStateChanged(final ItemEvent e) {
-                ((PlatformWithDeprecatedMethods) platform).setSleepAfterSync(e.getStateChange() == 1 ? 1000 : 0);
-            }
-        });
+        slowCheckbox.addItemListener(e -> {});
```

Comment:
Does it makes sense to add a listener that does nothing?

Reply:
removed